amplitude/amplitude_client.go

```diff
@@ -0,0 +1,91 @@
+package amplitude
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+)
+
+func newAmplitudeClient(
```

Comment:
Probably I need some background on this, I think `amplitudeClient` is a wrapped HTTP client, which is different from the SDK client in `client.go`. If yes, should we make it more clear with `amplitude_http_client` name?

Reply:
Renamed